### ISSUE TITLE
Maintained?

### ISSUE THREAD

**Issue by Frexuz** [apotonick/apotomo#160]
Last commit 2 years ago, and apotomo.de is down.

Will it be maintained in the future? Or should we stay away?
Anyone tried with Rails 5?

BTW: Asking because it looks awesome!

**Comment by apotonick** [comment]
It is awesome, thanks for asking! :joy: It's not maintained ATM and won't be, maybe in 2018.

**Comment by scarroll32** [comment]
Looks like a great approach

**Comment by mehmetaydogduu** [comment]
Hi, I can work together any developer which wants to maintain this gem.

**Comment by zapatify** [comment]
Has anyone tried this in Rails 5?  @Frexuz Did you ever try it in Rails 5?

**Comment by kg-currenxie** [comment]
Nope, never did.. Back then we went with https://github.com/antulik/pagelet_rails
Now, i think I would just do react :)

**Comment by zapatify** [comment]
LOL.  Yep.  We're contemplating what to do now.   Thanks.
